### ISSUE TITLE
Question: per_pixel_mean_subtract on test

### ISSUE THREAD

**Issue by Sirius083** [YixuanLi/densenet-tensorflow#19]
In Cifar10-densenet.py
Line 116: ds = dataset.Cifar10(train_or_test) 
Line 117: pp_mean = ds.get_per_pixel_mean()
Can the validation set use all test data statistics like per_pixel_mean?
Thanks in advance
